backend/src/tools/process_md.py

```diff
@@ -44,15 +44,15 @@ def process_md_docs(
 
     documents = load_docs(files_path=files_path)
 
-    with open("documents.txt", "w") as f:
-        for i, doc in enumerate(documents):
-            f.write(f"Doc ID: {i + 1}\n")
-            f.write(str(doc.page_content) + "\n")
-
     documents_knowledge_base = []
     for doc in documents:
+        try:
+            url = src_dict[doc.metadata["source"]]
+        except Exception:
+            url = ""
```

Comment:
Might be helpful to catch the specific exception and print out debug message

Reply:
Done. Added this now

Reply:
Where?